packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-modal-content.spec.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import JurisdictionModalContent from '../jurisdiction-modal-content';
+import { findByDisplayValue, render, screen } from '@testing-library/react';
+import RootStore from 'Stores/index';
+
+describe('JurisdictionModalContent', () => {
+    const mock_store = {
+        common: {},
+        client: {},
+        ui: {},
+    };
+    const mock_context = new RootStore(mock_store);
+    const mock_props = {
+        account_type: '',
+        is_virtual: false,
+        jurisdiction_selected_shortcode: '',
+        setJurisdictionSelectedShortcode: jest.fn(),
+        synthetic_available_accounts: [
+            {
+                market_type: 'gaming' as const,
+                name: '',
+                requirements: {
+                    after_first_deposit: {
+                        financial_assessment: [''],
+                    },
+                    compliance: {
+                        mt5: [''],
+                        tax_information: [''],
+                    },
+                    signup: [''],
+                },
+                shortcode: 'svg' as const,
```

Comment:
do we need this assertion as const

Reply:
yes cause we specify that the shortcode can only have values of `svg`, `bvi`, `labuan`, `malta_invest` and `vanuatu` if i didn't put it `as const`, typescript will check it as a string and will give typescript error stating that we are passing string instead of the fixed value. So `as const` helps typescript to define the value that we are trying to pass in is exactly the same value the component is accepting.

Reply:
can we use Juridction.svg here instead of passing 'svg'?

Reply:
sure let me change it 🙏